channel/consensus_channel/consensus_channel.go

```diff
@@ -9,16 +9,15 @@ import (
 	"github.com/statechannels/go-nitro/channel/state"
 	"github.com/statechannels/go-nitro/channel/state/outcome"
 	"github.com/statechannels/go-nitro/crypto"
-	"github.com/statechannels/go-nitro/protocols/directfund"
 
 	"github.com/statechannels/go-nitro/types"
 )
 
 type ledgerIndex uint
 
 const (
-	leader   ledgerIndex = 0
-	follower ledgerIndex = 1
+	Leader   ledgerIndex = 0
+	Follower ledgerIndex = 1
```

Comment:
Maybe:

```suggestion
	LeaderIndex   ledgerIndex = 0
	FollowerIndex ledgerIndex = 1
```

adding more context, after exporting.

Reply:
This would add more context for consuming packages.